P5/Source/Specs/spanGrp.xml

```diff
@@ -22,14 +27,72 @@ $Id$
   <classes>
     <memberOf key="att.global"/>
     <memberOf key="att.interpLike"/>
-    <memberOf key="att.typed" />
     <memberOf key="model.global.meta"/>
   </classes>
-  <content>
-    
-      <elementRef key="span" minOccurs="0" maxOccurs="unbounded"/>
-    
+  <content>    
+    <elementRef key="span" minOccurs="0" maxOccurs="unbounded"/>
   </content>
+  <attList>
+    <!--
+        The following <attDef> is a copy of the one in the
+        specifications of <interpGrp>, <span>, and <spanGrp>. It
+        *should* be part of the specification of att.interpLike rather
+        than in the specifications of the individual members of that
+        class. However, a bug in the Stylesheets means that won't work
+        at the moment. (See Stylesheets issue #370.)
+    -->
+    <attDef ident="type" usage="rec" mode="change">
```

Comment:
the other specs have `mode="replace"` here. This might work, but actually it should be `mode="add"` since we are not inheriting from a class (`att.typed`) anymore?

Reply:
Good question. Short answer is “idunno”. Will try to test over next few hours.

Reply:
- "add" causes duplicate definitions of `@type` (one with, one without the value list).
- "change" produces exactly the same results as "replace" **except** that the sample values list is dropped from the RELAX NG (and thus XSD) schema.
- Thus it seems "replace" is what we want.

Good catch, @peterstadler. Fixed in f0a2de56a.